### PR TITLE
Cut line for Three-Slice view

### PR DESCRIPTION
**Pull request by tdlinh2712**
This PR introduces the cutline, which is an indicator that shows the position of all three views in relation to each other. To enable it, click this button to show cutline on three-slice view.
cut-line-btn

### CODE REVIEW

**Review by JThomas2020 — APPROVED** [review]
Team 19

PR adds cut lines to visualise the relative positions of the slices on the 3-slice view. 
Toggle button works as expected. 
When the toggle is on and a point on one image is clicked/pressed, the cut lines move to where the mouse is clicked and the other two views update their slices to match the new positions of the cut lines. This feature only occurs when the user is both in the three window view and the toggle button is active (i.e. having the toggle active on single view and clicking wont adjust the slices of the other views).

**Review by Tamookk — APPROVED** [review]
Team 14

File: src/View/mainpage/DicomGraphicsScene.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    def mouseMoveEvent(self, event: QtWidgets.QGraphicsSceneMouseEvent) -> None:
+        if self.horizontal_view is not None and self.vertical_view is not None:
+            self.remove_cut_lines()
+            current_position = event.scenePos()
+            vertical_line_x = current_position.x()
+            horizontal_line_y = current_position.y()
+
+            self.add_cut_lines(vertical_line_x, horizontal_line_y)
+            self.update_slider(vertical_line_x, horizontal_line_y)
```
Comment (DuyTungHa) [inline src/View/mainpage/DicomGraphicsScene.py:81]: PEP-8 new line @minhhuyt

**Review by DuyTungHa — APPROVED** [review]

**Comment by tdlinh2712** [comment]
> Team 19
> 
> PR adds cut lines to visualise the relative positions of the slices on the 3-slice view.
> Toggle button works as expected.
> When the toggle is on and a point on one image is clicked/pressed, the cut lines move to where the mouse is clicked and the other two views update their slices to match the new positions of the cut lines. This feature only occurs when the user is both in the three window view and the toggle button is active (i.e. having the toggle active on single view and clicking wont adjust the slices of the other views).

Hi Thomas, that's an expected behaviour that Andrew mentioned in the done definition for this feature.

**Pull request merged**